site/tutorials/snippets/17/install/authCLI-istio.md

```diff
@@ -0,0 +1,37 @@
+## Connect your Keptn CLI to the Keptn installation
+
+In this section we are referring to the Linux/MacOS derivatives of the commands. If you are using a Windows host, please [follow the official instructions](https://keptn.sh/docs/0.14.x/operate/install/#authenticate-keptn-cli).
```

Comment:
The tutorial should probably point to the right Keptn version when linking to documentation:
```suggestion
In this section we are referring to the Linux/MacOS derivatives of the commands. If you are using a Windows host, please [follow the official instructions](https://keptn.sh/docs/0.17.x/operate/install/#authenticate-keptn-cli).
```